src/applications/vaos/components/ResidentialAddress.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export default function ResidentialAddress({ address }) {
```

Comment:
FYI for others: this is a copy of an existing component minus some facilities specific content, so we probably want to create an issue to consolidate the two components.